src/xss-common-qsort.h

```diff
@@ -113,6 +114,19 @@ struct zmm_vector;
 template <typename type>
 struct ymm_vector;
 
+namespace xss{
+namespace avx2{
+template <typename type>
+struct ymm_vector;
+
+inline arrsize_t replace_nan_with_inf(float *arr, int64_t arrsize);
+}
+}
+
+// key-value sort routines
+template <typename T1, typename T2>
+void avx512_qsort_kv(T1 *keys, T2 *indexes, int64_t arrsize);
```

Comment:
I had moved this definition to `src/avx512-64bit-keyvaluesort.hpp` We dont need this here.